src/sidebar/components/test/NotebookView-test.js

```diff
@@ -134,6 +145,41 @@ describe('NotebookView', () => {
     assert.isTrue(wrapper.find('NotebookFilters').exists());
   });
 
+  describe('synchronization of annotations', () => {
+    beforeEach(() => {
+      fakeStore.focusedGroup.returns({ id: 'hallothere', name: 'Hallo' });
+      fakeStore.pendingUpdateCount.returns(3);
+    });
+
+    it("doesn't display button to synchronize annotations if filters are applied", () => {
+      fakeStore.hasAppliedFilter.returns(true);
+      const wrapper = createComponent();
+
+      const button = wrapper.find('IconButton');
```

Comment:
For clarity if other buttons make it into this component, perhaps use an Enzyme property selector here to distinguish something distinctive about this `IconButton`?